templater/main.go

```diff
@@ -75,7 +74,24 @@ func main() {
 				}
 
 				if jobConfig.ImageBuild {
-					envVars = append(envVars, &types.EnvVar{Name: "ADDITIONAL_IMAGE_CACHE_REPOS", Value: "857151390494.dkr.ecr.us-west-2.amazonaws.com"})
+					if strings.Contains(jobConfig.JobName, "autoscaler") ||
```

Comment:
nit: Move this check to a `isCuratedPackagesPresubmit` method, and use it here and [above](https://github.com/aws/eks-anywhere-prow-jobs/pull/432/files#diff-eb41caacb1fedbfbd61bdb7d0a986a8a8548ae65b6107db1418844af5606e9acR84).